File: Aluno.py
class Aluno:
    def __init__(self, nome,  sobrenome):
        self.nome = nome
        self.sobrenome = sobrenome
    

    def nome_completo(self):
        n_c = self.nome + " " +self.sobrenome
        return n_c
    

luiz = Aluno('luiz','barboza')
print(luiz.nome_completo())

File: Aluno.js

 


function Aluno( n,  sn){
    this.nome = n
    this.sobrenome =sn
    

    this.nome_completo = function() {
        n_c = this.nome + " " +this.sobrenome;
        return n_c;
    }


}

luiz = new Aluno("Luiz","Barboza");
console.log(luiz.nome_completo())

